### PR TITLE
feat(voting-v2): make GAT a number of tokens rather than a percentage

### PR DESCRIPTION
**Pull request by mrice32**
**Motivation**

Computing GAT as a percentage of staked tokens can be dangerous if the number of staked tokens drops due to a migration, out-of-protocol incentive, or other extreme circumstance. It is preferable to have a GAT that is more stable and less dependent on the actions of users, either by tethering it to total supply or setting it to be a fixed number.

**Summary**

This PR proposes a slight change to the GAT system to simplify it and keep it stable by making it a flat number of tokens rather than a percentage.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [x]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

N/A

### CODE REVIEW

File: packages/core/contracts/oracle/implementation/VotingV2.sol [hunk 1]
```diff
@@ -782,12 +781,12 @@ contract VotingV2 is
     /**
      * @notice Resets the Gat percentage. Note: this change only applies to rounds that have not yet begun.
      * @dev This method is public because calldata structs are not currently supported by solidity.
-     * @param newGatPercentage sets the next round's Gat percentage.
+     * @param newGat sets the next round's Gat percentage.
```
Comment (md0x) [inline packages/core/contracts/oracle/implementation/VotingV2.sol:784]:
```suggestion
     * @param newGat sets the next round's Gat.
```

File: packages/core/contracts/oracle/implementation/VotingV2.sol [hunk 2]
```diff
@@ -782,12 +781,12 @@ contract VotingV2 is
...
      */
-    function setGatPercentage(uint256 newGatPercentage) public override onlyOwner {
-        require(newGatPercentage < 1e18);
-        gatPercentage = newGatPercentage;
-        emit GatPercentageChanged(newGatPercentage);
+    function setGat(uint256 newGat) public override onlyOwner {
+        require(newGat < votingToken.totalSupply() && newGat > 0);
```
Comment (md0x) [inline packages/core/contracts/oracle/implementation/VotingV2.sol:787]: I wonder if we would want to set a higher minimum, for example 5% of the totalSupply, or leave it completely open to the decision of the token holders.

**Comment by md0x** [comment]
Looks good to me!

File: packages/core/contracts/oracle/implementation/VotingV2.sol (hunk 2 above)
Comment (chrismaree) [reply packages/core/contracts/oracle/implementation/VotingV2.sol:787]: totally open. I dont think we should restrict this.

**Pull request merged**